### PR TITLE
add migration endpoints to swagger

### PR DESCRIPTION
**Pull request by blabadi**
**Description of changes**

**Type of Change**

- [ ] Bug
- [ ] Refactor
- [x] New Feature

**Checklist before requesting review:**

- [x] Check branch (code change PRs go to `develop` not master)
- [x] Manual testing
- [x] Regression tests completed and passing (double check number of tests).
- [x] Spelling has been checked.
- [x] Updated swagger docs accordingly (check it's still valid)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "local": "npm run build && sleep 3 && npm run watch-local",
     "resources": "copyfiles -u 1 src/resources/*.* dist/src",
     "serve-debug": "nodemon --inspect dist/src/server.js",
-    "watch-debug": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec node -r ts-node/register ./src/server.ts",
+    "watch-debug": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec node --inspect-brk -r ts-node/register ./src/server.ts",
```
Comment (jaserud) [inline package.json]: :+1:  right was gonna mention that break points don't seem to work without this...

**Comment by jaserud** [comment]
not sure why the `continuous-integration/jenkins/pr-merge` is saying fail... tests seem to be passing (tried it locally)

**Comment by blabadi** [comment]
> not sure why the `continuous-integration/jenkins/pr-merge` is saying fail... tests seem to be passing (tried it locally)

it's a flakey test, it's not consistent I suspect async thing not always working on time

**Comment by blabadi** [comment]
> > not sure why the `continuous-integration/jenkins/pr-merge` is saying fail... tests seem to be passing (tried it locally)
> 
> it's a flakey test, it's inconsistent I suspect async thing not always working on time

@jaserud line 95 in submission-to-clinical.ts is missing `await`

**Comment by jaserud** [comment]
Oh yea that would do it

File: package.json (hunk 1 above)
Comment (blabadi) [reply package.json]: yeah i just hated that it pauses at the first line

File: test/integration/submission/submission.spec.ts [hunk 2]
```diff
@@ -1206,7 +1206,7 @@ describe('Submission Api', () => {
         return err;
       }
 
-      return chai
+      return await chai
```
Comment (blabadi) [inline test/integration/submission/submission.spec.ts]: not really needed here, but won't harm

**Pull request merged**